src/lib/utils/snd_utils.c

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
So this needs to be modified. The Tinycompress project is dual licensed as LGPL and BSD. This license is incompatible to this project, so I can not consider this change. Please reconsider licensing as LGPL or BSD or both.

Reply:
Sure @vinodkoul , I will update the license.